src/components/CommentList.jsx

```diff
@@ -1,11 +1,31 @@
 import React, { useEffect, useState } from "react";
-import { getCommentsByArticleId } from "./utils/utils";
+import { getCommentsByArticleId, deleteComment } from "./utils/utils";
 import CommentCard from "./CommentCard";
 import "../../src/App.css";
 
-const CommentList = ({ article_id, comments, setComments, isCommentPosted }) => {
+const CommentList = ({
+  article_id,
+  comments,
+  setComments,
+  isCommentPosted,
+}) => {
   const [isLoading, setIsLoading] = useState(false);
   const [isError, setIsError] = useState(false);
+  const [deleteError, setDeleteError] = useState(false);
```

Comment:
Please set up a new state to be used to indicate that a comment is being deleted.

This state could be used to disable the 'delete' button for the time of deleting.